tests/connected_agents/packaged_host.py

```
@@ -0,0 +1,196 @@
+def run_packaged_host(
+    command: list[str],
+    *,
+    root: Path,
+    timeout: float = 5.0,
+    allowed_listeners: tuple[str, ...] = (),
+) -> PackagedHostResult:
+    environment = isolated_host_environment(root)
+    confined_command, network_isolation = _network_confined_command(
+        command, allowed_listeners=allowed_listeners
+    )
+    process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)